### PR TITLE
Events Page

### PR DESCRIPTION
**Pull request by kkaushal7**

### CODE REVIEW

File: frontend/src/styles/animations/Animation.scss [hunk 1]
```diff
@@ -0,0 +1,1307 @@
+@charset "UTF-8";
```
Comment (goyalritesh21) [inline frontend/src/styles/animations/Animation.scss]: Have you checked for existing classes for clashes?

File: frontend/src/styles/components/Events.scss [hunk 2]
```diff
@@ -0,0 +1,172 @@
+
```
Comment (goyalritesh21) [inline frontend/src/styles/components/Events.scss]: Check this file also with existing styles for clashes...

File: frontend/src/components/eventscard/Card.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from 'prop-types';
+
+import "../../styles/components/Events.scss";
+import "../../styles/animations/Animation.scss";
```
Comment (goyalritesh21) [inline frontend/src/components/eventscard/Card.js]: Do not import Styles here. Make them partials and add in src/styles/styles.scss

File: frontend/src/components/eventscard/Card.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      } = this.props;
+        return (
+            <Fragment>
+                <div
+                  data-wow-duration="1s"
+                  className="event fadeInDown"
+                >
+                  <div className="card1">
+                    <div className="card1-item card1-front">
+                      <img src={src} alt="cultural" />
```
Comment (goyalritesh21) [inline frontend/src/components/eventscard/Card.js]: Make the value of alt property dynamic. Here it is static having value cultural.

File: frontend/src/components/eventscard/Card.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                      <div className="headText">
+                        <p>{title}</p>
+                      </div>
+                    </div>
+                    <div className="card1-item card1-back">
+                      <img src={src} alt="cultural" />
```
Comment (goyalritesh21) [inline frontend/src/components/eventscard/Card.js]: Refer above comment at line 29

**Comment by kkaushal7** [comment]
OK,, i will see

**Pull request closed without merging**